packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -45,6 +45,8 @@ BuildRequires: rubygem(apipie-rails) >= 0.5.14
 BuildRequires: rubygem(fx) < 1.0
 BuildRequires: rubygem(pg)
 BuildRequires: rubygem(anemone)
+BuildRequires: rubygem(webrick)
+Requires: rubygem(webrick)
```

Comment:
Since this is only to patch anemone, I'd prefer to do it there. Also, this section will get overwritten by our auto-update so that would revert the changes. So if we keep it, it should be outside of the generated section.

Reply:
This was "intentional" so that the changes from https://github.com/Katello/katello/pull/10885 fix it "correctly" (that PR adds `webrick` as otherwise CI fails with the same error)

Reply:
Opened https://github.com/theforeman/foreman-packaging/pull/10411 to fix anemone directly